### PR TITLE
[TEVA-1283] Add database seed for review apps and local development

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1283

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,140 +1,187 @@
...
+#             family_name: "Joseph",
+#             given_name: "Hull")
+#
+# User.create(oid: "ED61B414-EFE4-4B32-82BC-FC9751F8443B",
+#             email: "mili.malde@digital.education.gov.uk",
+#             dsi_data: { "school_urns" => ["137138", "144523", "144522"], "school_group_uids"=>["16644"]},
+#             family_name: "Mili",
+#             given_name: "Malde")
+#
+# User.create(oid: "ED61B414-EFE4-4B32-82BC-FC9751F8443B",
```
Comment (cpjmcquillan) [inline db/seeds.rb:183]: duplicated

File: db/seeds.rb [hunk 2]
```diff
@@ -1,140 +1,187 @@
...
+                             easting: "228041",
+                             northing: "72116",
+                             geolocation: '(50.52350433336815,-4.427269703777728)',
+                             readable_phases: ["primary"],
+                             detailed_school_type: "Academy converter",
+                             school_type: "Academies",
+                             gias_data: { 'ReligiousCharacter (name)': 'None' },
+                             school_type: 'Academy')
+
+trust_1 = FactoryBot.create(:school_group,
```
Comment (cpjmcquillan) [inline db/seeds.rb:48]: can we use the `:trust` factory? Also can we seed a local_authority too?

Comment (cesidio) [reply db/seeds.rb:48]: sure

**Review by cpjmcquillan — APPROVED** [review]
Our names are all backwards.. but looks good to me 👍

**Comment by cesidio** [comment]
> Our names are all backwards.. but looks good to me 👍

adjusted, and put the names in alphabetical order

**Pull request merged**